docs/content/reference.md

```diff
@@ -111,6 +111,14 @@ The order in which the experiment will appear. Lower numbers appear first. Requi
 order: 0
 ```
 
+## `is_featured`
+
+Determines whether the experiment will be featured on the home page. Optional.
+
+```yaml
+order: 0
```

Comment:
Should this be `order: 0` or `is_featured: true`, or \<other\>?

Reply:
should be `is_featured`, I'll push a fix now